src/neuroglancer/ui/tool.ts

```diff
@@ -287,11 +297,12 @@ export class ToolBinder extends RefCounted {
     super.disposed();
   }
 
-  private deactivate() {
+  deactivate() {
     this.debounceDeactivate.cancel();
     const activation = this.activeTool;
     if (activation === undefined) return;
     this.activeTool = undefined;
+    activation.tool.deactivate();
```

Comment:
I just pushed 8ebac69881ab5a28d948daf743d6856d7a53c4db to eliminate the `Tool.deactivate` method since it was not previously used and is redundant.

Reply:
Makes sense, I've removed this part.